tuf/api/metadata.py

```diff
@@ -443,7 +443,7 @@ def verify_delegate(
                 key.verify_signature(delegated_metadata, signed_serializer)
                 signing_keys.add(key.keyid)
             except exceptions.UnsignedMetadataError:
-                logger.info("Key %s failed to verify %s", keyid, delegated_role)
+                logger.warning("Key %s failed to verify %s", keyid, delegated_role)
```

Comment:
warning is not required, debug is fine here (this just means one of the signing keys didn't sign the metadata: overall validity of the metadata will be handled two lines later)

Reply:
Done!